src/Groups/group_characters.jl

```diff
@@ -1724,6 +1724,28 @@ function trivial_character(G::GAPGroup)
     return class_function(G, [val for i in 1:Int(number_of_conjugacy_classes(G))])
 end
 
+@doc raw"""
+    regular_character(G::GAPGroup)
+
+Return the regular character of `G`.
+
+# Examples
+```jldoctest
+julia> G = symmetric_group(3);
+
+julia> values(regular_character(G))
+3-element Vector{QQAbElem{AbsSimpleNumFieldElem}}:
+ 6
+ 0
+ 0
+```
+"""
+function regular_character(G::GAPGroup)
+  ccl = conjugacy_classes(G)
```

Comment:
@ThomasBreuer is this correct? I saw that the other functions used `conjugacy_classes(character_table(G))`.

Reply:
OK, I read some docstring and understand why I need the character table